Archs/MIPS/MipsParser.cpp

```diff
@@ -58,6 +71,15 @@ const MipsRegisterDescriptor mipsPs2Cop2FpRegisters[] = {
 	{ L"vf30", 30 },	{ L"vf31", 31 },
 };
 
+const MipsRegisterDescriptor mipsRspCop0Registers[] = {
+	{ L"sp_mem_addr", 0 },	{ L"sp_dram_addr", 1 }, { L"sp_rd_len", 2 },
+	{ L"sp_wr_len", 3 },	{ L"sp_status", 4 },	{ L"sp_dma_full", 5 },
+	{ L"sp_dma_busy", 6 },	{ L"sp_semaphore", 7 },	{ L"dpc_start", 8 },
+	{ L"dpc_end", 9 },		{ L"dpc_current", 10 },	{ L"dpc_status", 11 },
+	{ L"dpc_clock", 12 },	{ L"dpc_bufbusy", 13 },	{ L"dpc_pipebusy", 14 },
+	{ L"dpc_tmem", 15 },
+};
+
```

Comment:
I tried it on real hardware (N64) now, and it seems like the high bit simply gets ignored by the RSP. In the CPU, accesses to invalid registers work without crashing as well. So all values should probably be considered valid, if possibly a bit nonsensical.